src/components/Button/Button.js

```diff
@@ -0,0 +1,9 @@
+import { StyledButton } from "./Button.styled";
+
+export const Button = ({ type, handleClick, children }) => {
```

Comment:
can we add prop types here?

Reply:
we can but I just forgot to delete this component, it was for testing the form only. The callback functions are passed to `Modal` component. I will delete it now.